src/kirby/components/modal/services/modal.helper.ts

```diff
@@ -4,21 +4,65 @@ import { Animation } from '@ionic/core';
 
 import { ModalConfig } from '../modal-wrapper/config/modal-config';
 import { ModalWrapperComponent } from '../modal-wrapper/modal-wrapper.component';
+import { KirbyAnimation } from '@kirbydesign/designsystem/animation/kirby-animation';
+import { ModalConfigHelper } from '../modal-wrapper/config/modal-config.helper';
 
 @Injectable()
 export class ModalHelper {
   constructor(private ionicModalController: IonicModalController) {}
 
   public async showModalWindow(
-    config: ModalConfig,
+    conf: ModalConfig,
     registerModal: (modal: { close: (data?: any) => {} }) => void
   ): Promise<any> {
+    const config: ModalConfig = {
```

Comment:
Consider extracting to separate function, e.g. `MapConfig` or `MergeDefaultConfig`
Also consider using [object literal spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) or [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) to simplify mapping.

Reply:
Seperated into fuction, but since there are some logic involved spread and assign did not make it any better I think.